README.md

```diff
@@ -1,5 +1,19 @@
 # toolbox - bring your own tools with you
 
+The purpose of this fork is to extend toolbox for use as a main environment on openSUSE MicroOS. Long-term, it may be extended to Fedora Silverblue, Ubuntu Core, or other read-only rootfs operating systems. The script currently supports deb and rpm based distros in the container. It should be distro-agnostic regarding the host.
+
+**Fork goals: Replace installing software for the user on MicroOS with a new command that snapshots the container (podman commit), runs through a script to bring in icon and desktop files, and cleans up old snapshots. Currently a script exists to loop through every installed package and search for relevant ones and modifying them to work and be identified with a container. A cleanup script needs to be added for container deletion. Possibly implement container deletion as a new option to toolbox.**
+
+Still needs implementation:
+
+    1. Snapshots of containers
+
+    2. An easy method of restoring a container to a snapshot
+
+    3. Autotomatic snapshot management
+
+    4. A less hacky approach to keeping track of installed packages
+
```

Comment:
I don't understand your "Fork" description. Why should this be a fork? Could it be that this belongs to your own fork?

Reply:
Right! So, @spacemule , thanks for your work on improving toolbox. I have the feeling that it could be quite useful, especially for the MicroOS Desktop use-case. Thanks also for the pull request, so that we can review your work and try to integrate it (or as much of it as it makes sense to integrate) in the main project.

And the parts that we'll manage to integrate in the official repository, won't be part of a fork any longer. Therefore, this comment should not be among the pieces that we pick from your patches and integrate in this repository, as this is the official version of the tool and not a fork. :-)

Hope I made things clear enough... If not, do not hesitate to ask.

Reply:
Thanks for being so patient with me. When I started this, it was a personal project, so I didn't think about how it would merge back, and being unfamiliar with using git, I just updated this description so anyone coming to my fork would understand what I'm working on. I did not intend for this to make it back to y'all.